.github/labeler.yml

```diff
@@ -0,0 +1,25 @@
+build:
+  - .github/**/*
+
+dependencies:
+  - buildSrc/**/*
+
+documentation:
+  - docs/**/*
+  - README.adoc
+  - LICENSE
+  - CHANGELOG.adoc
+  - CODE-OF-CONDUCT.adoc
+  - CONTRIBUTION.adoc
+  - readmeOSS.adoc
```

Comment:
Maybe I overlook something here - shouldn’t be `readmeOSS.adoc` located at the root dir?

Reply:
Solved by: https://github.com/d4l-data4life/hc-fhir-sdk-kmp/pull/30